presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -207,15 +207,16 @@ void checkSplitsForBatchTask(
 }
 
 struct ZombieTaskStats {
-  const std::string taskId;
-  const std::string taskInfo;
-
-  explicit ZombieTaskStats(const std::shared_ptr<exec::Task>& task)
-      : taskId(task->taskId()), taskInfo(task->toString()) {}
-
-  std::string toString() const {
-    return SystemConfig::instance()->logZombieTaskInfo() ? taskInfo : taskId;
-  }
+  const std::string info;
+  const long numExtraReferences;
+
+  ZombieTaskStats(
+      const std::shared_ptr<exec::Task>& task,
+      long numExtraReferences_)
```

Comment:
Comment say the extra references could be either from velox or presto? Thanks!

Reply:
This is conveyed on another level.
The structure has no knowledge about it.